src/features/user/MFV2/TpoProjects/index.tsx

```diff
@@ -77,7 +77,6 @@ export default function ProjectsContainer({ profile }: Props) {
                   >
                     <ProjectSnippet
                       project={project.properties}
-                      editMode={false}
                       displayPopup={true}
                     />
```

Comment:
There's a warning here as `displayPopup` is not an accepted prop. Is `displayPopup` needed?